instrumentation/opentelemetry-instrumentation-openai/tests/test_chat_completions.py

```diff
@@ -0,0 +1,162 @@
+import json
+
+import pytest
+
+from opentelemetry.semconv._incubating.attributes import (
+    gen_ai_attributes as GenAIAttributes,
+)
+
+
+@pytest.mark.vcr()
+def test_chat_completion(exporter, openai_client):
+    llm_model_value = "gpt-4"
+    messages_value = [
+        {"role": "user", "content": "Say this is a test three times"}
+    ]
+
+    kwargs = {
+        "model": llm_model_value,
+        "messages": messages_value,
+        "stream": False,
+    }
+
+    response = openai_client.chat.completions.create(**kwargs)
+    spans = exporter.get_finished_spans()
+    chat_completion_span = spans[-1]
```

Comment:
since we are looking at the last, maybe add a comment above why (my guess is we want to ignore http spans, and they are being added by default)

Reply:
Are we sure these are http spans? Last time I've seen more spans that I expected it was me not removing the instrumentation in `_uninstrument`.